apps/language_server/lib/language_server/providers/hover.ex

```diff
@@ -114,16 +114,24 @@ defmodule ElixirLS.LanguageServer.Providers.Hover do
     end
   end
 
-  defp dep_name(root_mod_name, project_dir) do
+  defp dep_name(subject, project_dir) do
+    root_mod_name = root_module_name(subject)
+
     if not elixir_mod_exported?(root_mod_name) do
       ""
     else
       s = root_mod_name |> source()
 
       cond do
-        third_dep?(s, project_dir) -> third_dep_name(s, project_dir)
-        builtin?(s) -> builtin_dep_name(s)
-        true -> ""
+        third_dep?(s, project_dir) ->
+          %ElixirSense.Location{file: source} = ElixirSense.definition(subject, 1, 1)
```

Comment:
Please handle a cace when location is nil

Reply:
fixed